common/buildcraft/transport/pipe/behaviour/PipeBehaviourObsidian.java

```diff
@@ -191,18 +191,22 @@ public long receivePower(long microJoules, boolean simulate) {
             return microJoules;
         }
         EnumFacing openFace = getOpenFace();
-        for (int d = 1; d < 5; d++) {
```

Comment:
It would probably be cleaner to do the same as the above if statement: do "if openFace == null return microJoules" rather than wrapping everything in an additional indent level.